<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferenceDialog</class>
 <widget class="QDialog" name="PreferenceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>186</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabGeneral">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <widget class="QFrame" name="frameRecordPath">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>371</width>
         <height>101</height>
        </rect>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QRadioButton" name="radioDefaultRecordPath">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>140</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Default</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioCustomizedRecordPath">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>30</y>
          <width>140</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Customized</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelRecordPath">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>70</y>
          <width>301</width>
          <height>31</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="text">
         <string>.</string>
        </property>
       </widget>
       <widget class="QPushButton" name="buttonRecordPath">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>70</y>
          <width>41</width>
          <height>31</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>Choose customized path</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelRecordDirectory">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>191</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>User data directory:</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="tabSecurity">
      <attribute name="title">
       <string>Security</string>
      </attribute>
      <widget class="QFrame" name="framePassword">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>9</y>
         <width>371</width>
         <height>41</height>
        </rect>
       </property>
       <widget class="QLabel" name="labelPassword">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>3</y>
          <width>231</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Account password: </string>
        </property>
       </widget>
       <widget class="QPushButton" name="buttonChangePassword">
        <property name="geometry">
         <rect>
          <x>231</x>
          <y>0</y>
          <width>141</width>
          <height>40</height>
         </rect>
        </property>
        <property name="text">
         <string>Modify</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
